blaze/compute/tests/test_mongo_compute.py

```diff
@@ -268,30 +269,35 @@ def test_datetime_handling(events):
                        events)) == set([2, 3])
 
 
+@xfail(sys.platform == 'win32', reason='on win32, results are backwards')
```

Comment:
we can odo this to a set so order doesn't matter. That can be another change though,